docs/api/source/index.rst

```diff
@@ -1,13 +1,102 @@
 :orphan:
 
-==============================
-solid-client-access-grants API
-==============================
+====================================================
+@inrupt/solid-client-access-grants API Documentation
+====================================================
 
+``@inrupt/solid-client-access-grants`` is a JavaScript library for
+requesting and managing access given to an agent for a resource. These
+access grants are signed by a Solid Pod owner’s private key, which
+itself is managed by the Solid Pod server.
```

Comment:
```suggestion
requesting and managing access given to an agent for a resource. These
access grants are represented by Verifiable Credentials, signed by an Issuer associated to the Pod server where the resources are hosted.
```
I'm afraid the notion of Pod owner could be confusing here: the private key is managed by the Pod server's operator, and not really associated to the resource owner.

Reply:
oh! Good call, we need to update the README.md too then!